packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec

```
@@ -73,6 +73,9 @@ install -m 0644 .%{gem_instdir}/config/%{plugin_name}.yml \
 %{gem_instdir}/test
 
 %changelog
+* Thu May 23 2024 Partha Aji - 4.14-0.1.nightly
```

```suggestion
* Thu May 23 2024 Partha Aji - 1.14.0-0.1.pre.master
```

and you gotta reset `%release`

---

and it was already done in #10822 so you can leave it out here